ProcessMaker/processmaker
# Add 'Vendor Onboarding' default template

## Issue & Reproduction Steps
Ticket: [FOUR-7950](https://processmaker.atlassian.net/browse/FOUR-7950)

Add Vendor Onboarding default template

## Solution
- Create migration to add Vendor Onboarding as a default template

## How to Test
1. Run `php artisan migrate` 
2. A new Vendor Onboarding process should be listed under the Templates tab with the category 'Default Templates'
3. This template should be editable
4. A new process should be able to use this template


[FOUR-7950]: https://processmaker.atlassian.net/browse/FOUR-7950?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
